akka-diagnostics/src/main/scala/akka/diagnostics/ConfigChecker.scala

```diff
@@ -795,6 +792,47 @@ class ConfigChecker(system: ExtendedActorSystem, config: Config, reference: Conf
       else Nil
     }
 
+  private def checkRemoteDeployment(): List[ConfigWarning] =
+    ifEnabled("remote-prefer-cluster") { checkerKey =>
+      val path = "akka.actor.provider"
+      if (config.getString(path) == "remote" || config.getString(path) == "akka.remote.RemoteActorRefProvider")
+        warn(checkerKey, path, "remote is possible, but prefer cluster") ////TODO add better description?
+      else Nil
+    }
+
+  private def checkCreateActorRemotely(): List[ConfigWarning] =
+    ifEnabled("create-actor-remotely") { checkerKey =>
+      val path = """akka.actor.deployment."/...".remote""""
+
+      val isRemoteDeployment = config
+        .getConfig("""akka.actor.deployment""")
+        .withoutPath("default")
+        .entrySet()
+        .toArray()
+        .exists { case x: Entry[String, ConfigValue] =>
+          x.getKey.matches("""^\"\/.*\"\.remote""")
+        } // is of the type "/...".remote
+      if (isRemoteDeployment)
+        warn(
+          checkerKey,
+          path,
+          "Deploying an actor remotely is deprecated and not supported. As per https://doc.akka.io/docs/akka/current/remoting.html#creating-actors-remotely"
+        ) //TODO add better description?
+      else Nil
+    }
+
+  private def checkClusterWatchFailureDetector(): List[ConfigWarning] =
+    ifEnabled("cluster-watch-failure-detector") { checkerKey =>
+      val path = "akka.remote.watch-failure-detector"
+      if (isClusterConfigAvailable) {
+        warn(
+          checkerKey,
+          path,
+          "Remote watch failure detector shouldn't be used when cluster is used" // TODO explain why?
```

Comment:
should be enough pointing to `### Remote Watch Failure Detector` in `config-checker.md`?

Reply:
using suggested message `Some features, such as remote watch, will be unsafe when using remote without Akka Cluster.`